package.json

```diff
@@ -167,16 +167,11 @@
                     "command": "azureFunctions.restartFunctionApp",
                     "when": "view == azureFunctionsExplorer && viewItem == azureFunctionsFunctionApp",
                     "group": "2@3"
-                },
-                {
-                    "command": "azureFunctions.deployZip",
-                    "when": "view == azureFunctionsExplorer && viewItem == azureFunctionsFunctionApp",
-                    "group": "3@1"
                 }
             ],
             "explorer/context": [
                 {
-                    "command": "azureFunctions.deployZip",
+                    "command": "azureFunctions.deploy",
                     "when": "explorerResourceIsFolder == true",
                     "group": "zzz_@1"
```

Comment:
Just curious, but what's with the ground name 'zzz'?

Reply:
It makes sure its the last command in the context menu alphabetically (since this is the 'folder' context menu and we don't control that menu)